### ISSUE TITLE
Email handling problem after upgrade to Magento 2.3.3

### ISSUE THREAD

**Issue by richardreen** [magento/magento2#25171]
### Preconditions (*):

Magento 2.3.3
Php 7.2

After upgraded to Mangeto 2..3.3  (old was 2.3.2) we have problems with email handling.
We have now 2 problems:

1. All email send to customers is empty but the content of the email is placed in an attachment called attachment.html. 

2. In backend the preview of the email templates is empty. When we look at the sources we see that the inline component ( where to content of the template is placed) is empy. Even a new loaded templated has this problem.

We have no error messages in apache log of magento system.log

In Magento 2.3.2 we had no problems.

#
### Steps to reproduce (*)

1. Upgrade from Magento 2.3.2 to 2.3.3
2. Place order and wait for succes email. 
3. Go to backend and under marketing email...do preview of current template in use.
4. Make new template and load one of the temples...do preview...

###Expected result

Normal email says your order succesfull and resume of order. No attachment
Preview of template should be filled with content o current or  loaded template..

### Actual result (*)
 Email is empty...with an attachment.
Preview of templates is empty.

**Comment by richardreen** [comment]
@magento give me 2.3-develop instance - upcoming 2.3.x release

**Comment by magento-engcom-team** [comment]
Hi @richardreen. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @richardreen, here is your [Magento instance](https://i-25171-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25171-2-3-develop.instances.magento-community.engineering/admin](https://i-25171-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by richardreen** [comment]
on this vanilla instance it is working normal, ik can see a preview...so i will investigate our install, another extension is intervering in 2.3.3..

**Comment by coquito696** [comment]
I am having the same exact situation. Email becomes an attachment and email itself is empty. What resolved the problem? Removed extension?

**Comment by JannPadley** [comment]
I also have exactly the same problem after upgrade to 2.3.3

Following make no difference:
a) disable extensions
b) disable Google reCAPTCHA
c) enable developer mode
d) disable cache
e) disable DKIM signing for non-smtpd messages (postfix main.cf)

But I found a quick and dirty fix:
/vendor/magento/module-email/Model/Transport.php
Change line 93 as follows:
before fix: $zendMessage = Message::fromString($this->message->getRawMessage())->setEncoding('utf-8');
after fix: $zendMessage = Message::fromString($this->message->getRawMessage());

I only half understand why this works. My fix does not alter the message content, but the message headers are different and seem to be the cause of the problem:
2.3.2: Content-Transfer-Encoding: 8bit
2.3.3 before fix: Content-Transfer-Encoding: quoted-printable, Content-Disposition: =?utf-8?Q?inline?=
2.3.3 after fix: Content-Transfer-Encoding: quoted-printable, Content-Disposition: inline

Have I discovered a bug, whereby the values of headers are incorrectly UTF encoded in 2.3.3?
This is a bit strange, because Transport.php has not changed significantly since 2.3.2.
I suppose that something has changed in Magento Mail Framework or Zend.
I don't know whether my fix creates a security risk, and allows the exploits that 2.3.3 blocks.
I hope this helps, and would love to hear comments from more knowledgeable coders...
